marlowe-playground-client/src/Simulation.purs

```diff
@@ -275,7 +269,11 @@ transactionComposer ::
 transactionComposer state =
   div [ classes [ ClassName "transaction-composer", ClassName "composer" ] ]
     [ ul [ class_ (ClassName "participants") ]
-        [ transaction state isEnabled ]
+        ( if (Array.null pendingInputs) then
+            [ text "Empty transaction" ]
+          else
+            [ transaction state isEnabled ]
+        )
```

Comment:
You can replace these parenthesis with the square brackets and then you don't need them inside. Also I think the ones in the condition of the if are not necessary